.github/workflows/mdbook.yml

```diff
@@ -74,6 +72,23 @@ jobs:
         with:
           source-dir: ./book/
 
+      - name: Remove preview
+        if: github.event_name == 'pull_request' && github.event.action == 'closed'
```

Comment:
Do you need the corresponding `!= closed` in the action above?

Reply:
No, that action handles the `closed` event. It just doesn't remove the `.nojekyll` file. So, I _could_ remove just the `.nojekyll` in this step and it would work. I'm doing the entire folder just to be future proof.